src/Tasks.UnitTests/ReadLinesFromFile_Tests.cs

```diff
@@ -17,29 +16,33 @@
 
 namespace Microsoft.Build.UnitTests
 {
-    sealed public class ReadLinesFromFile_Tests
+    public sealed class ReadLinesFromFile_Tests
```

Comment:
In the future, please don't mix this kind of style change with real code changes. It makes the PR much harder to review. In this repo, you're welcome to make these changes, but please put them in individual commits, so the "real" changes can be easily identified.

Reply:
Hm.. not sure anymore if I did that as real action. It could come from hitting `Ctrl+K, Ctrl+D` (Format Document) and I expected that the solution has somewhere a editorconfig that keeps those thing align. Isn't that the case?

Reply:
Yeah, it's probably the case that it came from a format-document command. We haven't done a cleanup of the repo to define and match our standards in . . . a very long time.

Reply:
It would be nice to have that, so that anyone like me hitting this shortcut (which is fairly automated on my fingers) will not get into troubles ;)